apps/mark-scan/frontend/src/pages/admin_screen.tsx

```diff
@@ -88,129 +115,75 @@ export function AdminScreen({
-                  try {
-                    await setPrecinctSelectionMutation.mutateAsync({
-                      precinctSelection: newPrecinctSelection,
-                    });
-                  } catch {
-                    // Handled by default query client error handling
-                  }
-                }}
-                election={election}
-                mode={
-                  pollsState === 'polls_closed_final' ||
-                  election.precincts.length === 1
-                    ? 'disabled'
-                    : 'default'
-                }
-              />
-              <br />
-              <Caption>
-                Changing the precinct will reset the Ballots Printed count.
-              </Caption>
-              {election.precincts.length === 1 && (
-                <React.Fragment>
-                  <br />
-                  <Caption>
-                    Precinct cannot be changed because there is only one
-                    precinct configured for this election.
-                  </Caption>
-                </React.Fragment>
-              )}
-            </P>
-            <H6 as="h2">Ballot Mode</H6>
-            <P>
-              <SegmentedButton
-                label="Ballot Mode"
-                hideLabel
-                onChange={() =>
-                  setTestModeMutation.mutate({ isTestMode: !isTestMode })
-                }
-                options={[
-                  { id: 'test', label: 'Test Ballot Mode' },
-                  { id: 'official', label: 'Official Ballot Mode' },
-                ]}
-                selectedOptionId={isTestMode ? 'test' : 'official'}
-              />
-              <br />
-              <Caption>
-                Switching the mode will reset the Ballots Printed count.
-              </Caption>
-            </P>
-          </React.Fragment>
-        )}
-        <H6 as="h2">Date and Time</H6>
-        <P>
-          <Caption>
-            <CurrentDateAndTime />
-          </Caption>
-        </P>
-        <P>
-          <SetClockButton logOut={() => logOutMutation.mutate()}>
-            Set Date and Time
-          </SetClockButton>
+        <H2 as="h1">Election Manager Menu</H2>
+        <P>Remove the election manager card to leave this screen.</P>
+        <P style={{ fontSize: '1.2em' }}>
+          <Font weight="bold"> Ballots Printed: </Font>{' '}
+          {format.count(ballotsPrintedCount)}
         </P>
-        <H6 as="h2">Configuration</H6>
+        <H3 as="h2">Configuration</H3>
+        {election.precincts.length > 1 && (
+          <P>
+            <ChangePrecinctButton
+              appPrecinctSelection={appPrecinct}
+              updatePrecinctSelection={updatePrecinctSelection}
+              election={election}
+              mode={
+                pollsState === 'polls_closed_final' ? 'disabled' : 'default'
+              }
+            />
+          </P>
+        )}
```

Comment:
Ok, that sounds reasonable to me. Another option would be to just show the precinct as text, but given that it's already in the election info bar I think it's ok.

Related: it would be nice if the election info was nearer to the unconfigure button in general so the action was grouped with the thing it's modifying. But that's a bigger puzzle